content/articles/2019-06-29-getting-done-for-next-release.md

```diff
@@ -0,0 +1,79 @@
+Title: Getting done for next release
+Date: 2019-06-29 07:00
+Category: GSOC
+Tags: GSOC, GSOC19, poliastro
+slug: 2019-06-29-getting-done-for-next-release
+lang: en
+Author: Jorge Martínez Garrido
+
+## Version 0.13 is getting closer
+
+Version 0.13 is expected to be tagged in about a month. As any other release,
+it includes new features such as a new method in the `Orbit` class called
+`change_attractor` or the `trail` parameter in `StaticOrbitPlotter`. However,
+this release will also include really important bug fixes such as:
+
+* Minor issues related to Lambert's problem.
+* Propagator's are supposed not to hang out anymore due to robust solutions and propagation methods.
+
+Now that my final exams ended I am completely free to work in poliastro and
+direct all my efforts to the software. I want to close issues
+[#495](https://github.com/poliastro/poliastro/issues/495) and
+[#475](https://github.com/poliastro/poliastro/issues/475)  in the next days since they are previously commented bugs.
+
+### Propagator methods in the twobody problem
+
+Kepler's equation (KE) allows us to solve either for the time at a given
+position or the position for a known time. Although it is a very short
+equation, there have been lots of numerical methods along with history since it
+has not an analytical solution.
+
+The most famous numerical method related to root finding is probably the so
+called [Newton-Raphson
+method](https://es.wikipedia.org/wiki/M%C3%A9todo_de_Newton). By making use of
+the function derivative it is possible to reach with great accuracy and speed
+the solution to the equation. Other methods also use higher order derivatives
+to achieve a faster connvergence. Although the Newton-Raphson methods works
+really well most of the cases, it is possible that sometimes the derivative at
+some point of the evaluated function becomes zero, making to diverge the
+numerical method.
+
+Along the years, different approaches have been developed to solve this famous
+equation: conversion to a third order polynomial, higher order derivatives
+iterative methods, series expansion... Most of the different papers available
+claim convergence regarding the speed of computation, somehting that is not
+*fair* if we consider that CPUs computational power evolve as years pass.
+Numerical methods should be compared fixing a relative error and number of
+iterations. But remember: there is not a perfect numerical methods. While for
+examples the *bisection* method always converges it does really slowly, on the
+other hand, *Newton-Rapshon* based methods have a great rate convergence but
+are subjected to zero derivative singularities.
+
+After a huge research on the topic, the most interesting algorithms from my
+point of view are the following ones:
+
+* [Fukushima's work on KE](https://www.researchgate.net/project/Solution-of-Kepler-Equations)
```

Comment:
Do you have a DOI for this article by any chance?

Reply:
Oh, it is a link to a set of papers, not sure if there exists a DOI for that kind of links :confused:

Reply:
Let's use http://dx.doi.org/10.1007/BF00049384 then, which is his most cited article on the matter according to the list. I don't like ResearchGate very much, and also having the DOI makes it easier to obtain the article by... other means :smiling_imp:

Reply:
Me too, I do not feel comfortable with that page either. For me looks kind of social-media research place :sweat_smile: